src/components/Chat/Chat.scss

```diff
@@ -0,0 +1,112 @@
+@import '../../assets/css/colors';
+$border: 1px solid #f9d342;
+$box-shadow: 2px 2px 3px #999;
+$border-radius: 50px;
+$position: fixed;
+$margin-top: 18px;
+$margin-top-delete: 17px;
+$width: 60px;
+$height: 60px;
+$bottom: 40px;
+$right: 40px;
+$bottom-btn: 16px;
+$right-btn: 33px;
+$font-size: 13px;
+$font-size-floating-chat: 25px;
+
+.Chat {
+  .my-float {
```

Comment:
CSS has a similar built-in `float`, it can be good if you rename this class in another way